### DISCUSSION TITLE
Error when playing a video with the screen off

### DISCUSSION THREAD

**Discussion by pmmo18** [libre-tube/LibreTube#1777] [Q&A] [answered]
When I lock the phone and the audio is playing in the background, it cuts off automatically after 5 minutes, I unlock the phone and the video is paused.
Is there any way that it doesn't pause every 5 minutes and I can finish listening to the entire video?
Thanks for the help! <3

**Comment by Bnyro** [comment, marked as answer]
I recommend you click the `Audio` button which is part of the player controls before you turn your screen off which will keep the video play on in the background. Otherwise Android will automatically "pause" or "freeze" the app to save battery and hence the video stops.
